dll/win32/oleacc/CMakeLists.txt

```diff
@@ -29,15 +32,14 @@ set_source_files_properties(oleacc.rc PROPERTIES OBJECT_DEPENDS "${oleacc_rc_dep
 
 add_library(oleacc MODULE
     ${SOURCE}
-    guid.c
+    ${PCH_SKIP_SOURCE}
     oleacc.rc
-    ${CMAKE_CURRENT_BINARY_DIR}/oleacc_classes_p.c
-    ${CMAKE_CURRENT_BINARY_DIR}/oleacc_stubs.c
+    ${CMAKE_CURRENT_BINARY_DIR}/proxy.dlldata.c
```

Comment:
What makes the dlldata file different from the other stubs (for which we disable PCH)?